feature-configs/base/sriov/07-ipv6-network_attachment_defintion.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: openshift.io/sriovnic
+  name: dual-stack-net-attachment-def
+  namespace: ipv6namespace
```

Comment:
I thought it would actually be nicer to have the ipv6 related pods (and attachment) in a separate namespace.
This works fine. If you insist on having it all in one namespace I'll change it.

Reply:
I don't insist, but if we leave a different namespace then we need also another `SriovNetwork` pointing to the ipv6 namespace.